packages/core/aqua-to-js/src/generate/interfaces.ts

```diff
@@ -117,12 +119,19 @@ export class TSTypeGenerator implements TypeGenerator {
     const serviceDecl = `service: ${srvName}Def`;
     const serviceIdDecl = `serviceId: string`;
 
-    const registerServiceArgs = [
-      [serviceDecl],
-      [serviceIdDecl, serviceDecl],
-      [peerDecl, serviceDecl],
-      [peerDecl, serviceIdDecl, serviceDecl],
-    ];
+    const registerServiceArgs =
+      // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
+      (srvDef.defaultServiceId as DefaultServiceId).s_Some__f_value != null
```

Comment:
Can you assert the type here first? `as` doesn't check the type, if I understand correctly.

Reply:
And some doc comments would be great here.

Reply:
Added some comments. No need to assert here. If types are incorrect assertion wont give any benefit here. Tests should catch runtime error here if this wrong type will be fixed later.